🐛 Fix UV search not working

kirkkwang opened the pull request:
# Story

- ref https://github.com/scientist-softserv/utk-hyku/issues/439

[🐛 Update IIIF Print to fix OCR search](https://github.com/scientist-softserv/utk-hyku/pull/440/commits/801bc8fd276a6111ad7bff0971c3a7bcb5de9be6) 

This commit will update IIIF Print which has a fix for the UV OCR search
not working.

# Expected Behavior Before Changes

UV OCR search would not return anything

# Expected Behavior After Changes

UV OCR search should now return the correct responses

# Screenshots / Video

<img width="1184" alt="image" src="https://github.com/scientist-softserv/utk-hyku/assets/19597776/e02ec6f1-9fd4-459c-9263-a43c4cb18dda">

ShanaLMoore commented on lib/iiif_print/data/fileset_helper_decorator.rb:
```diff
@@ -0,0 +1,14 @@
+# OVERRIDE IIIF Print v1.0.0 to deal with Attachment ids
+
+IiifPrint::Data::FilesetHelper.module_eval do
+  def first_fileset
```
Should this change be done in iiif print now that we discovered a similar issue in ADL?

kirkkwang replied:
Yes, you're right.  Also Jeremy did say it's safe to update IIIF Print since the Rodeo stuff isn't enabled, I'll make a PR for that.

Commit:
🐛 Update IIIF Print to fix OCR search

This commit will update IIIF Print which has a fix for the UV OCR search
not working.
